### PR TITLE
Add enrichment to strip HTML tags

### PR DESCRIPTION
**Pull request by no-reply**
Closes https://www.pivotaltracker.com/story/show/85082098

### CODE REVIEW

File: spec/lib/krikri/enrichments/strip_html_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe Krikri::Enrichments::StripHtml do
+  it_behaves_like 'a field enrichment'
+
+  let(:start_value) { '<html>Moomin <i><b>Valley</i></b>' }
```
Comment (no-reply) [inline spec/lib/krikri/enrichments/strip_html_spec.rb]: You have me worried about unicode, now. Checking in on it here.

**Comment by anarchivist** [comment]
I'm :+1: if there are no changes or concerns about Unicode.

**Comment by no-reply** [comment]
The latest commit confirms unicode is okay, and also fixes up test duplication.

I'd prefer to keep the two commits separate, barring objections.

**Comment by anarchivist** [comment]
No objections from me. @markbreedlove and @AudreyAltman?

**Comment by AudreyAltman** [comment]
:+1: I like what you've done with the shared examples specs.

**Comment by anarchivist** [comment]
:hospital:

**Pull request merged**